2020-01-Winter/4_trees/README.md

```diff
@@ -94,22 +129,87 @@ Go to [Solution](#s3)   [Top](#top)
 <!-- Don't remove -->
 <a name="s1"/>
 
-### 1. SOLUTION 1 TODO :bug:
+### 1. Path Sum
 
-Source: TODO :bug:
+Source: [Leetcode]https://leetcode.com/problems/path-sum/
 
 #### Naive/Simple Solution
-
-TODO :bug:
+This implementation uses the queue for BFS iterating over the nodes.
+**Time Complexity**: O(n) where N is the number of nodes in the tree.
+
+class Solution {
```

Comment:
Without triple backticks for code block formatting, the code gets rendered as follows:
![code](https://user-images.githubusercontent.com/29528527/74424026-1528ff00-4e06-11ea-893e-5d6d924b052b.png)

Include `c++` as the language after the opening backtick section so you get syntax highlighting on top of code block formatting.
````suggestion
```c++
class Solution {
````